src/main/java/org/gitlab4j/api/models/Variable.java

```diff
@@ -14,6 +14,8 @@
     private String value;
     @JsonProperty("protected")
     private Boolean isProtected;
+    @JsonProperty("masked")
+    private Boolean isMasked;
```

Comment:
@PhilippeVienne 
No need to name this ```isMasked```, just name it ```masked``` and remove the ```@JsonProperty("masked")``` line.

The reason ```isProtected``` was used is because ```protected``` is a reserved keyword in Java.